dashboard/src/main/resources/templates/component.ftl

```diff
@@ -1,14 +1,17 @@
 <!DOCTYPE html>
 <html lang="en-US">
   <#include "macros.ftl">
+  <#assign groupId = artifact.getGroupId()>
+  <#assign artifactId = artifact.getArtifactId()>
+  <#assign version = artifact.getVersion()>
   <head>
     <meta charset="utf-8" />
-    <title>Google Cloud Platform Dependency Analysis Report for ${groupId}:${artifactId}:${version}</title>
+    <title>Google Cloud Platform Dependency Analysis Report for ${artifact}</title>
```

Comment:
revert; it's unclear what toString does here

Reply:
Makes sense. Reverted.